assets/css/style.css

```diff
@@ -0,0 +1,136 @@
+/* font styling */
+h1,
+h2,
+h3,
+h4,
+p,
+ul,
+li {
+font-family: 'Open Sans', sans-serif;
```

Comment:
This line could be easier to apply globally with something like:

```css
html, body {
  font-family: 'Open Sans', sans-serif;
}
```

This will apply to all of the elements you've listed above, with less typing! You can also set a default font size, then within each tag (like `p`) you can overwrite those defaults.